test/e2e/lua/dynamic_configuration.go

```diff
@@ -156,11 +156,29 @@ var _ = framework.IngressNginxDescribe("Dynamic Configuration", func() {
 
 		Expect(nginxConfig).ShouldNot(Equal(newNginxConfig))
 	})
+
+	It("should set sticky cookie correctly", func() {
+		resp, _, errs := gorequest.New().
+			Get(f.IngressController.HTTPURL).
+			Set("Host", "foo.com").
+			End()
+
+		Expect(len(errs)).Should(BeNumerically("==", 0))
+		Expect(resp.StatusCode).Should(Equal(http.StatusOK))
+		Expect(resp.Header.Get("Set-Cookie")).Should(ContainSubstring("SERVERID="))
+		Expect(resp.Header.Get("Set-Cookie")).Should(ContainSubstring("Expires="))
+		Expect(resp.Header.Get("Set-Cookie")).Should(ContainSubstring("Max-Age=172800"))
```

Comment:
please move this test to annotations/affinity.go

Reply:
OK, after the way to handle the expires annotation is confirmed, I will do this